dsaps/models.py

```diff
@@ -51,26 +53,37 @@ def get_record(self, uuid, rec_type):
 
     def filtered_item_search(self, key, string, query_type,
                              selected_collections=''):
+        """Search using filtered item endpoint."""
         offset = 0
         items = ''
         item_links = []
         while items != []:
-            endpoint = f'{self.url}/rest/filtered-items?'
+            endpoint = f'{self.url}filtered-items?'
             params = {'query_field[]': key, 'query_op[]': query_type,
                       'query_val[]': string, '&collSel[]':
                       selected_collections, 'limit': 200, 'offset': offset}
-            logger.info(params)
-            print(endpoint)
             response = requests.get(endpoint, headers=self.header,
                                     params=params, cookies=self.cookies)
-            print(f'Response url: {response.url}')
+            logger.info(f'Response url: {response.url}')
             response = response.json()
             items = response['items']
             for item in items:
                 item_links.append(item['link'])
             offset = offset + 200
         return item_links
 
+    def post_coll_to_comm(self, base_url, comm_handle, coll_name):
+        """Post collection to specified community."""
+        endpoint = f'{self.url}handle/{comm_handle}'
+        community = requests.get(endpoint, headers=self.header,
+                                 cookies=self.cookies).json()
+        comm_id = community['uuid']
+        collection = json.dumps({'name': coll_name})
```

Comment:
In the requests call below change `data=collection` to `json=collection` and just pass it the dictionary directly.

Reply:
That is very cool